extra-requirements.txt

```diff
@@ -6,3 +6,7 @@ scipy ~= 1.4.1
 sympy ~= 1.5.1
 torch ~= 1.5.0
 mypy ~= 0.770
+sphinx ~=3.1.2
```

Comment:
i'm not sure, but maybe we want to create yet another requirements file that is only necessary for building the docs? what if i'd like to have all extra requirements (sympy, torch, etc) but don't want to build the docs?

Reply:
yes, good point. Implemented that.

Reply:
e91c89152ecadf38e62f2bd16c437cf4b9cace0a

Reply:
Also refined the instructions for installation: 96f232e9c727be072bd43fde5fb8793cef9b49da

Reply:
and here: 2b53c21a4afd66e6e6cce6b1c8593e9a7db692b7